packages/nitro-utils/package.json

```diff
@@ -39,7 +39,6 @@
     "access": "public"
   },
   "dependencies": {
-    "@sentry/types": "8.37.1",
     "@sentry/utils": "8.37.1"
```

Comment:
We might need `@sentry/core` here because of https://github.com/getsentry/sentry-javascript/pull/14431

Reply:
Ah yea, good catch!